pkg/common/moerr/error.go

```diff
@@ -32,71 +32,50 @@ const (
 	WARN    = 2
 
 	// Group 1: Internal errors
-	ERROR_START              = 1000
-	INTERNAL_ERROR           = 1001
-	NYI                      = 1002
-	ERROR_FUNCTION_PARAMETER = 1003
+	ERROR_START              = 100
```

Comment:
Do you want to give it a type MOErrorCode to these constants?

Reply:
done